packages/create-pantheon-decoupled-kit/src/generators/umami.generator.ts

```diff
@@ -0,0 +1,28 @@
+import type { CustomActionConfig } from 'node-plop';
+import type { DecoupledKitGenerator } from '../types';
+
+export const umami: DecoupledKitGenerator = {
+	name: 'umami-demo',
+	description: 'Umami profile data and components',
```

Comment:
```suggestion
	description: 'Drupal's Umami profile data and components add-on for the next-drupal starter',
```
The description could be a bit more precise. I am now realizing that there is no place to see these descriptions... Maybe they should show up with the list of generators? You don't need to implement this in this ticket, just thinking out loud. Thoughts?